actionmailer/lib/action_mailer/railtie.rb

```diff
@@ -25,7 +26,9 @@ class Railtie < Rails::Railtie # :nodoc:
       options.cache_store ||= Rails.cache
 
       if options.show_previews
-        options.preview_path ||= defined?(Rails.root) ? "#{Rails.root}/test/mailers/previews" : nil
+        options.preview_paths |= ["#{Rails.root}/test/mailers/previews"]
+      else
+        options.preview_paths.clear
```

Comment:
What's this for? It doesn't look like it's covered by tests.

Reply:
👍 Yes, this is actually not needed.